crates/node-bindings/src/nodes/reth.rs

```diff
@@ -142,12 +149,14 @@ pub struct Reth {
 impl Reth {
     /// Creates an empty Reth builder.
     ///
-    /// The mnemonic is chosen randomly.
-    pub const fn new() -> Self {
+    /// The instance number is set to a random number between 1 and 200 by default to reduce the
+    /// odds of port conflicts. This can be changed with [`Reth::instance`]. Set to 0 to use the
+    /// default ports. 200 is the maximum number of instances that can be run set by Reth.
+    pub fn new() -> Self {
         Self {
             dev: false,
             block_time: None,
-            instance: 0,
+            instance: rand::thread_rng().gen_range(1..200),
```

Comment:
hmm, there's currently no way to set the port, right?
the `Geth` type has his and allows port 0 so that the os can choose a random port and it should get parsed from cli output.
could we do the same here?

Reply:
you can set the instance to roughly control the port mapping but it indeed would be better to have that fully configurable as well

added the ability to customize the ports

Reply:
you can now define `.http_port(0)` (same for other ports) to use a random port picked by `reth`